test/cmd/apps.sh

```diff
@@ -65,7 +65,7 @@ run_daemonset_history_tests() {
   kubectl apply -f hack/testdata/rollingupdate-daemonset.yaml --record "${kube_flags[@]}"
   kube::test::wait_object_assert controllerrevisions "{{range.items}}{{$annotations_field}}:{{end}}" ".*rollingupdate-daemonset.yaml --record.*"
   # Rollback to revision 1 - should be no-op
-  kubectl rollout undo daemonset --to-revision=1 "${kube_flags[@]}"
+  kubectl rollout undo daemonset --all --to-revision=1 "${kube_flags[@]}"
```

Comment:
Please don't modify existing tests, but rather create a new one verifying this is working as expected.

Reply:
Ok

Reply:
Ok

Reply:
Hmm, so, actually i didn't changed the behavior of the test here, since this PR changes behavior of the complete command i just used command as it should be used now.
OLD `kubectl rollout undo daemonset` == NEW `kubectl rollout undo daemonset --all`

Reply:
Right, I see what you're saying. In that case please do:
1. fix these tests to use actual daemon set name
2. add a new tests where `--all` will actually verify it's applied to more than a single resource to ensure your all is actually selecting all, currently this PR doesn't do it.

Reply:
Sounds good, will do.